src/frontend/src/lib/services/request-poh-credential.services.ts

```diff
@@ -0,0 +1,82 @@
+import {
+	INTERNET_IDENTITY_ORIGIN,
+	POH_ISSUER_CANISTER_ID,
+	POH_ISSUER_ORIGIN
+} from '$lib/constants/app.constants';
+import { i18n } from '$lib/stores/i18n.store';
+import { userProfileStore, type UserProfile } from '$lib/stores/settings.store';
+import { toastsError } from '$lib/stores/toasts.store';
+import { Principal } from '@dfinity/principal';
+import { isNullish, nonNullish } from '@dfinity/utils';
+import {
+	requestVerifiablePresentation,
+	type VerifiablePresentationResponse
+} from '@dfinity/verifiable-credentials/request-verifiable-presentation';
+import { get } from 'svelte/store';
+
+// This credential type is defined by the issuer Decide AI
+const POH_CREDENTIAL_TYPE = 'ProofOfUniqueness';
+
+export const requestPohCredential = async ({
+	credentialSubject
+}: {
+	credentialSubject: Principal;
+}): Promise<boolean> => {
+	const i18Copy = get(i18n);
```

Comment:
I know that code comes from II just by reading `i18Copy` 😉.

We mostly directly inline access `get(i18n)` or we destruct what we need.

Reply:
Hehe, I destructured it.